showcase/package.json

```diff
@@ -97,6 +98,7 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-qunit": "^8.1.1",
     "loader.js": "^4.7.0",
+    "luxon": "^3.4.2",
```

Comment:
thought: I don't think we need this either; `luxon` is available via the components library